server/src/test/java/io/crate/action/sql/SessionTest.java

```diff
@@ -265,6 +265,20 @@ public void testExtractTypesFromInsertFromQuery() throws Exception {
         assertThat(parameterTypes, is(new DataType[]{DataTypes.STRING}));
     }
 
+    @Test
+    public void test_foo() throws Exception {
```

Comment:
Maybe there is a better name than foo for this test method ?

Reply:
Oh, I didn't intend to commit these changes with the fix. Will remove it